config.py

```diff
@@ -0,0 +1,8 @@
+import os
+basedir = os.path.abspath(os.path.dirname(__file__))
+
+class Config(object):
+  SECRET_KEY = os.environ.get('SECRET_KEY') or 'this-key-is-super-secret'
```

Comment:
This seems like a way to have an accidentally dangerous exploit. Could we throw a warning or something if it's not available?

Reply:
Yes. I'll do that.